### ISSUE TITLE
Getting ImportError from performing `from rdkit.DataStructs import cDataStructs`

### ISSUE THREAD

**Issue by seankala** [rdkit/rdkit#2867]
**Configuration:**

- RDKit Version:  '2019.09.1'
- Operating system: Ubuntu 16.04
- Python version (if relevant): 3.7.5
- Are you using conda? Yes.
- If you are using conda, which channel did you install the rdkit from? conda-forge
- If you are not using conda: how did you install the RDKit?

**Description:**

When I run my code I get the following error message:

```Python
ImportError: /usr/lib/x86-64-linux-gnu/libstdc++.so.6: version 'CXXABI_1.3.11' not found (required by /home/user1/anaconda3 ... )
```

The full traceback is:

```Python
Traceback (most recent call last):
  File "./main.py", line 4, in <module>
    from train import Trainer
  File "/home/user1/repository/train.py", line 12, in <module>
    from utils import make_batches
  File "/home/user1/repository/utils.py", line 9, in <module>
    from rdkit import Chem, DataStructs
  File "/home/user1/anaconda3/envs/user1conda/lib/python3.7/site-packages/rdkit/Chem/__init__.py", line 18, in <module>
    from rdkit import DataStructs
  File "/home/user1/anaconda3/envs/user1conda/lib/python3.7/site-packages/rdkit/DataStructs/__init__.py", line 13, in <module>
    from rdkit.DataStructs import cDataStructs
ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.11' not found (required by /home/user1/anaconda3/envs/user1conda/lib/python3.7/site-packages/rdkit/DataStructs/../../../../libRDKitDataStructs.so.1)
```

I've tried to run `conda update libgcc`, `conda install gxx_linux-64`, but neither have worked. Would anybody be able to point me in the correct direction? Thanks in advance.

```
Your code sample here
```

**Comment by greglandrum** [comment]
Hi @seankala, this looks like a problem with your configuration. I just tried a fresh miniconda install (v4.7.12) on a new ubuntu 16.04 docker image and everything worked fine.
The one thing I can think of is that you might have LD_LIBRARY_PATH set and this is messing with conda. Can you check that?

**Comment by seankala** [comment]
Hello @greglandrum thanks for the reply. I actually got RDKit to work again. I'm not sure exactly what was wrong, but my assumption is that I had recently undergone an operation of switching servers and therefore something went wrong in the process. I just completely deleted Anaconda and related packages and reinstalled them again. `LD_LIBRARY_PATH` was set correctly to the default path, but following the error message I had set it to `/home/user1/anaconda3/lib`, which did not work either.
